src/app/_components/tabs/file/vulnerability.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from 'react';
+
+import type { TVulnerability } from '@/agents/audit';
+
+import { CircleDot, Loader2 } from 'lucide-react';
+import Link from 'next/link';
+
+import { Button } from '@/components/ui/button';
+import { Textarea } from '@/components/ui/textarea';
```

Comment:
nit: TextArea

Reply:
That's the default export from shadcn/ui. Wontfix :P